dungeon/doc/quickstart.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Dungeon: Quickstart"
+---
+
+Diese Anleitung beschreibt Schritt für Schritt, wie Dungeon: Learning by Questing heruntergeladen, konfiguriert und gestartet werden kann.
+
+## Schritt 1: Herunterladen der Ressourcen 
```

Comment:
Warum magst Du die Checklisten nicht?

Reply:
Checklisten machen irgendwie nur Sinn, wenn man auch wirklich was "abcheckt", oder nicht? 
In diesem Fall ist das ja mehr ne Anleitung. 

Wenn du Checkboxen willst, gerne.

Dann für jeden Header eine? Oder für jeden Sub-Bulletpoint? Oder beides? 



PS: Heut ich auch irgendwie eine schwere Geburt. Keine Ahnung warum. Sorry für "unnötigen" Mehraufwand auf deiner Seite.

Reply:
Hab checklisten raus gemacht